src/pages/patch/patchTabs/tasks/TasksTable.tsx

```diff
@@ -34,14 +36,21 @@ export const TasksTable: React.FC<Props> = ({ data, columns }) => {
     }
   };
 
+  const transformData = (tasks: Tasks) =>
+    tasks.map((task) => {
+      const t = { ...task, children: undefined };
```

Comment:
I think we should remove `children: undefined` and create a return interface for transformData. that way we don't have to make this type alias `as TaskResult[]` which misrepresents the result of `transformData`

Reply:
Nice catch this is actually no longer needed

Reply:
It was from before i found the prob that lets your rename the children column name